### DISCUSSION TITLE
Welcome to rusti-cal Discussions!

### DISCUSSION THREAD

**Discussion by arthurhenrique** [arthurhenrique/rusti-cal#24] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by arthurhenrique** [comment]
https://github.com/arthurhenrique/rusti-cal/pull/21

**Comment by debajyoti1990** [comment]
Hello, any plan for porting in windows ?

**Reply by arthurhenrique** [reply]
Hello, I didn't considered porting for windows, because I don't work with it. By the way, I think would be great to have windows supporting. Thx

**Reply by debajyoti1990** [reply]
Not blaming you, actually not everyone gets to work with linux machine in workplace even if they want to 😐. Thanks.
